js/tests/unit/tooltip.js

```diff
@@ -680,6 +680,42 @@ $(function () {
     $styles.remove()
   })
 
+  test('should not be adjust position when scrolling', function () {
```

Comment:
Yeah, my comment was directed at @cvrebert, should've added his handle to the post.